docs/markdown/android/avatar.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Avatar
+description: Avatar is used to represent a user. Every Avatar image has a subtle color wash.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/af/75/6f/af756f3ad767857e5695d6cb626c1a12.png" alt="all variations of the avatar component"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="Do" />
+      - To reflect a person, company or brand within the product
+  </Group>
+  <Group>
+  <Dont title="Don't" />
+     - To represent a group of people, companies and/or brands. use [AvatarGroup](https://gestalt.pinterest.systems/web/avatargroup) instead
```

Comment:
Didn't notice this in the iOS docs, both in both pages these should be "When to use" and "When not to use"